doc/classes/TouchScreenButton.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TouchScreenButton" inherits="Node2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Button for touch screen devices.
 	</brief_description>
 	<description>
+		Button that can be visible on all or only on touch screen devices.
```

Comment:
This sentence needs attention.  Needs work, but something like:

Button for touch screen devices. You can set it to be visible on all screens, or only on touch devices.